test/voronoi-test.js

```diff
@@ -1,6 +1,6 @@
-import tape from "@observablehq/tape";
-import Delaunay from "../src/delaunay.js";
-import Context from "./context.js";
+const tape = require("tape-await");
+const {Delaunay} = require("../");
+const Context = require("./context.js");
```

Comment:
This effectively changes the tests to run against the built version rather than the source, right? Asking just in case it wasn't intentional. One disadvantage of this is when you develop something continuously, you have to rerun the build constantly instead of having instant feedback.

Reply:
Ah, yes. It was intentional to normalize to what happens in the other D3 modules, but I didn't realize the implication. If it's a drag, we should find a better solution.

Reply:
A variant here https://github.com/d3/d3-delaunay/pull/115

Reply:
In the old way, the build was happening implicitly on the fly because this module uses esm. The other D3 modules don’t currently use esm; they only use Rollup. And the thinking was it’s better to test the built bundle because that’s what consumers of the library will be using (and historically there have been some bugs introduced by bundlers, but that’s less of a problem these days).

I think it’d be great to switch to native support for ES modules but we’re not quite there yet. I think I’m ambivalent about whether it’s worth the trouble to switch all the D3 modules to use esm for testing or to run Rollup before tests. Rollup should be fast enough for a small library that the difference is negligible.